integration_tests/integration/prison-register-details.spec.ts

```diff
@@ -8,21 +8,21 @@ context('Prison register - prison details navigation', () => {
     cy.task('reset')
     cy.task('stubLogin')
     cy.task('stubAuthUser')
-    cy.task('stubGetPrisonsWithFilter', [albanyPrison])
-    cy.task('stubGetPrison', albanyPrison)
+    cy.task('stubGetPrisonsWithFilter', [albanyPrison, moorlandPrison])
+    cy.task('stubGetPrison', moorlandPrison)
```

Comment:
Using Moorland as have set Albany Prison with no address and Moorland with an address